config.go

```diff
@@ -1077,6 +1089,19 @@ type OCSPConfig struct {
 	ResponderOverrides map[string]string
 }
 
+// CertificateEventData contains contextual information for
+// an obtained, renewed or revoked certificate.
+type CertificateEventData struct {
+	// Domain or subject name of the certificate.
+	Name string
+
+	// Storage key for the issuer used for this certificate.
+	IssuerKey string
```

Comment:
```suggestion
	// The issuer used to manage this certificate (if applicable).
	Issuer Issuer
```

Might as well include the actual value, then the user can get the key as well as do type assertions, etc; more useful.

Reply:
As I said earlier, this would require a copy, because this will likely get fired via a goroutine. It's not safe to pass the actual issuer. See https://github.com/caddyserver/certmagic/pull/150#issuecomment-956557862

Reply:
I think it's safe if it's read-only, yeah? And while it's true we don't think people should be calling Issue(), it's possible that type-assertions could be useful to callers.

In any case, I don't care too strongly on this. Leaving it as a string is fine if that's best.